scripts/fiber/tests-passing-except-dev.txt

```diff
@@ -5,150 +5,18 @@ src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js
 * should suggest property name if available (ssr)
 
 src/renderers/dom/shared/__tests__/ReactDOMServerIntegration-test.js
-* renders a blank div with client render on top of bad server markup
-* renders a div with inline styles with client render on top of bad server markup
-* renders a self-closing tag with client render on top of bad server markup
-* renders a self-closing tag as a child with client render on top of bad server markup
-* renders simple numbers with client render on top of bad server markup
-* renders simple strings with client render on top of bad server markup
-* renders string prop with true value with client render on top of bad server markup
```

Comment:
I guess all of these should test these particular values can issue warnings, but in reality they all just test that the tag name didn't match. Which is why this fixes all of them.